docs/pr_template.md

```diff
@@ -0,0 +1,33 @@
+# Frontend Pull Request
+
+Description of Changed/Added Behaviour
+---
+
+_Short description of the behaviour that was added or modified._
+
+Motivation and Context
+---
+_Information about the wider context and future plans of the changes._
+
+Link to Related Ticket
+---
+_If the task has e.g. a JIRA ticket linked to it, this is the place to add it._
```

Comment:
Jira -> Linear, although the Linear is not visible to OS contributors. Does it make sense to have this section here>

Reply:
Yes, but I haven't changed this file, I think I moved it from one location to another. Will update it though.